Public/Src/Cache/ContentStore/Library/Logging/Logger.cs

```diff
@@ -221,7 +231,12 @@ public void Log(Severity severity, string message)
         /// <inheritdoc />
         public void LogFormat(Severity severity, string messageFormat, params object[] messageArgs)
         {
-            LogString(severity, string.Format(CultureInfo.InvariantCulture, messageFormat, messageArgs));
+            if (messageArgs != null && messageArgs.Length > 0)

```

Comment:
The code is duplicated 3 times. I think it makes sense to extract a helper.

Like FormatStringIfNeeded.

Reply:
Turns out: if I do this, our tests start timing out with resource usage exceeded. I think it should be left as-is